### PR TITLE
Revisions to "Project Plan" Sections of the Proposal

### PR DESCRIPTION
**Pull request by FSXAC**
Revised the sections under "Project Plan" section. In particular here are some of the common edits I've made:

- Refactored bullet points into paragraphs.
- Reduced the wordiness of sentences.
- Reduced redundant information
- Updated `proposal.tex` with labels for referencing

### CODE REVIEW

**Comment by FSXAC** [comment]
[Edit "Project plan" section](https://trello.com/c/IUlT6vLS/46-edit-project-plan-section)

File: docs/proposal/3_3_major_responsibilities.tex [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
-\begin{itemize}
-\item Be available to meet in-person or online given reasonable notice (72 hours)
-\item Provide additional education/training on machine learning and neural networks, if required
-\item Provide additional financial support, if required (see budget section)
-\end{itemize}
\ No newline at end of file
+The client is responsible for being available to meet in-person or online
+given a reasonable notice of one week. They are expected to provide
+(reference to) necessary education or training material. They are also expected
+to provide additional financial supprot if needed (see section \ref{budget} for
```
Comment (peterdeutsch) [inline docs/proposal/3_3_major_responsibilities.tex]:
```suggestion
to provide additional financial support if needed (see section \ref{budget} for
```

File: docs/proposal/3_5_schedule.tex [hunk 2]
```diff
@@ -1,4 +1,10 @@
 \subsubsection{Major Schedule}
+
+FIXME: we should add a timeline chart with labels of milestone I, II, and III
```
Comment (peterdeutsch) [inline docs/proposal/3_5_schedule.tex]: I don't know if a literal chart is necessary, but I agree that a timeline is important.

File: docs/proposal/3_5_schedule.tex [hunk 3]
```diff
@@ -7,6 +13,10 @@ \subsubsection{Major Schedule}
 
 
 \subsubsection{Weekly Schedule}
+
+TODO: also make a weekly schedule outlining our availablity (or lack of, due
```
Comment (peterdeutsch) [inline docs/proposal/3_5_schedule.tex]: Keep in mind, that we don't have much prior knowledge of our course loads (especially in 2nd term), so I'm not entirely sure how confident we can be here. Perhaps brief platitudes here.

File: docs/proposal/rev_history.tex [hunk 4]
```diff
@@ -8,7 +8,8 @@ \section*{Revision History}
 \hline
 Version \# & Initials & Release Date & Changes Made \\ \hline
 0.0 & MH & 2019-09-30 & Initial skeleton of the document.\\
-1.0 & MW & 2019-10-08 & First draft\\
+1.0 & MW & 2019-10-08 & First draft.\\
+1.0.1 & MH & 2019-10-09 & Revised ``Project Plan'' section.\\
```
Comment (peterdeutsch) [inline docs/proposal/rev_history.tex]: We should probably establish our versioning strategy ASAP. For drafts like this one I think we can stick at v1.0 (or better yet imo, 0.1).

**Pull request merged**